src/createServer.js

```diff
@@ -1,3 +1,37 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const serverError = require('./serverError').serverError;
+const convertToCase = require('./convertToCase/convertToCase').convertToCase;
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    const normalizedURL = new URL(req.url, 'http://localhost:5700');
+
+    res.setHeader('Content-Type', 'application/json');
+
+    const reqText = normalizedURL.pathname.slice(1);
+    const reqCase = normalizedURL.searchParams.get('toCase');
+
+    const errors = serverError(reqText, reqCase);
+
+    if (errors.errors.length) {
+      res.end(JSON.stringify(errors));
+    } else {
+      const { originalCase, convertedText } = convertToCase(reqText, reqCase);
+
+      const resultConvert = {
+        originalCase,
+        targetCase: reqCase,
+        originalText: reqText,
+        convertedText,
+      };
+
+      res.end(JSON.stringify(resultConvert));
```

Comment:
You forgot about the response status and text:
You should respond with status 200 and OK status text.